buildchow/templates/product-template.yaml

```diff
@@ -117,39 +109,44 @@
             - maven.test.error.ignore=true
             - download.cache.directory={download-cache}
             - targetPlatformGroup=jbdevstudio
-            - jbosstools-site=http://download.jboss.org/jbosstools/updates/nightly/core/${{jbosstools_site_stream}}/
+            - jbosstools-site=http://download.jboss.org/jbosstools/mars/snapshots/builds/jbosstools-build-sites.aggregate.site_${{jbosstools_site_stream}}/latest/all/repo/
           pom: ${{WORKSPACE}}/sources/product/site/pom.xml          
       - maven-target:
-          goals: "org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
-          maven-version: maven-3.x
+          goals: "-U org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
```

Comment:
You cannot seriously argue for not having actual releases used in our builds, can you ?

Why don't we just move back to using latest version from master ? 

Because then we can't actually trust our builds.

What in these releng scripts are *ever* expected to change ? if there are anything in them that requires changes between releases we have failed to actually make these reliable.